streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -0,0 +1,174 @@
+    ResourceConfig resourceConfig = ConfigCache.getOrCreate(ResourceConfig.class, jobConfig);
+    if (resourceConfig.isTaskCpuResourceLimit()) {
+      resourceMap.put(ResourceConfig.RESOURCE_KEY_CPU, resourceConfig.taskCpuResource());
+    }
+    if (resourceConfig.isTaskMemResourceLimit()) {
+      resourceMap.put(ResourceConfig.RESOURCE_KEY_MEM, resourceConfig.taskMemResource());
+    }
+    return resourceMap;
+  }
+
+  private List<ExecutionVertex> createExecutionVertics() {
+    List<ExecutionVertex> executionVertices = new ArrayList<>();
+    for (int index = 1; index <= parallelism; index++) {
+      executionVertices.add(new ExecutionVertex(jobVertexId, index, this));
+    }
+    return executionVertices;
+  }
+
+  public Map<Integer, RayActor<JobWorker>> getExecutionVertexWorkers() {
+    Map<Integer, RayActor<JobWorker>> executionVertexWorkersMap = new HashMap<>();
+
+    Preconditions.checkArgument(
+        executionVertexList != null && !executionVertexList.isEmpty(),
+        "Empty execution vertex.");
+    executionVertexList.stream().forEach(vertex -> {
+      Preconditions.checkArgument(
+          vertex.getWorkerActor() != null,
+          "Empty execution vertex worker actor.");
+      executionVertexWorkersMap.put(vertex.getVertexId(), vertex.getWorkerActor());
+    });
+
+    return executionVertexWorkersMap;
+  }
+
+  public String getJobName() {
+    return jobName;
+  }
+
+  public int getJobVertexId() {
+    return jobVertexId;
+  }
+
+  public String getJobVertexName() {
+    return jobVertexName;
+  }
+
+  public JobVertex getJobVertex() {
+    return jobVertex;
+  }
+
+  public int getParallelism() {
+    return parallelism;
+  }
+
+  public Map<String, String> getJobConfig() {
+    return jobConfig;
+  }
+
+  public Map<String, Double> getResources() {
+    return resources;
+  }
+
+  public List<ExecutionVertex> getExecutionVertexList() {
+    return executionVertexList;
+  }
+
+  public void setExecutionVertexList(
+      List<ExecutionVertex> executionVertex) {
+    this.executionVertexList = executionVertex;
+  }
+
+  public List<ExecutionJobEdge> getOutputEdges() {
+    return outputEdges;
+  }
+
+  public void setOutputEdges(
+      List<ExecutionJobEdge> outputEdges) {
+    this.outputEdges = outputEdges;
+  }
+
+  public List<ExecutionJobEdge> getInputEdges() {
+    return inputEdges;
+  }
+
+  public void setInputEdges(
+      List<ExecutionJobEdge> inputEdges) {
+    this.inputEdges = inputEdges;
+  }
+
+  public StreamProcessor getStreamProcessor() {
+    return streamProcessor;
+  }
+
+  public LanguageType getLanguageType() {
+    return jobVertex.getLanguageType();
+  }
+
+  public VertexType getVertexType() {
+    return jobVertex.getVertexType();
+  }
+
+  public long getBuildTime() {
+    return buildTime;
+  }
+
+  public boolean isSourceVertex() {
+    return getVertexType() == VertexType.SOURCE;
+  }
+
+  public boolean isProcessVertex() {
+    return getVertexType() == VertexType.PROCESS;
```

Comment:
What's meaning about VertexType.PROCESS?
May `Transformation` or `Transform` be better in view of abstraction.

Reply:
This is because ```VertexType``` 's definition. Can we unify this to 'Transformation'? @chaokunyang

Reply:
Use 'TRANSFORMATION' instead of 'PROCESS' in ```VertexType```.